build_tools/build_libtorch.sh

```diff
@@ -68,29 +68,35 @@ build_pytorch() {
   fi
 
   BUILD_SHARED_LIBS=ON \
+  BUILD_CAFFE2_OPS=OFF \
+  INTERN_BUILD_ATEN_OPS=OFF \
+  ATEN_NO_TEST=OFF \
+  USE_LITE_INTERPRETER_PROFILER=OFF \
   BUILD_TEST=OFF \
   GLIBCXX_USE_CXX11_ABI=1 \
   CMAKE_OSX_ARCHITECTURES=${CMAKE_OSX_ARCHITECTURES} \
+  MACOSX_DEPLOYMENT_TARGET=${MACOSX_DEPLOYMENT_TARGET} \
   INTERN_BUILD_ATEN_OPS=OFF \
   INTERN_DISABLE_ONNX=ON \
   INTERN_USE_EIGEN_BLAS=ON \
   MAX_JOBS=${MAX_JOBS} \
   ONNX_ML=OFF \
   USE_BREAKPAD=OFF \
   USE_CUDA=OFF \
+  USE_ITT=OFF \
   USE_DISTRIBUTED=OFF \
   USE_EIGEN_FOR_BLAS=OFF \
-  USE_FBGEMM=OFF \
+  USE_FBGEMM=ON \
   USE_GLOO=OFF \
-  USE_KINETO=OFF \
+  USE_KINETO=ON \
   USE_MKL=OFF \
   USE_MKLDNN=OFF \
   USE_MPS=OFF \
   USE_NCCL=OFF \
   USE_NNPACK=OFF \
   USE_OBSERVERS=OFF \
   USE_OPENMP=OFF \
-  USE_PYTORCH_QNNPACK=OFF \
+  USE_PYTORCH_QNNPACK=ON \
   USE_QNNPACK=OFF \
   USE_XNNPACK=OFF \
   ${PYTHON_BIN} setup.py  bdist_wheel -d "$WHEELHOUSE"
```

Comment:
```suggestion
  ${PYTHON_BIN} setup.py  bdist_wheel -d "$WHEELHOUSE" | grep -Ev '^(adding|copying)'
```
to silence all of the `adding` and `copying` lines

Reply:
Will add in a follow on commit so we don't have to rerun CI